src/main/java/dev/slimevr/vr/processor/HumanPoseProcessor.java

```diff
@@ -210,17 +211,25 @@ public void setLegTweaksEnabled(boolean value) {
 	public void setFloorClipEnabled(boolean value) {
 		if (skeleton != null) {
 			skeleton.setFloorclipEnabled(value);
-			this.getSkeletonConfig().saveToConfig(server.config);
-			server.saveConfig();
+			server
```

Comment:
this wont work. 
You either need to call server.getConfigManager().saveConfig() after that.
Or call ``this.getSkeletonConfig().save()`` then ``server.getConfigManager().saveConfig()``